### PR TITLE
fix: upgrade dataplex tables to managed, create new zone, remove manual table creation

### PR DESCRIPTION
**Pull request by bradmiro**
This PR covers several changes to the codebase:

- Separating the data into three dataplex zones: one raw and two curated
- Creating a second staging bucket to separate image (raw) data from table (curated) data
- Upgrading Dataplex assets to managed, programmatically (This feature is currently absent from the Terraform provider. FIled a FR with the eng team)
- Parallelize the object copy step
- Reorder Terraform graph to run Workflow after provisioning Dataplex

### CODE REVIEW

File: workflows.tf [hunk 1]
```diff
@@ -78,3 +100,47 @@ resource "google_workflows_workflow" "project_setup" {
...
+  depends_on = [
+    google_workflows_workflow.project_setup,
+    google_dataplex_asset.gcp_primary_textocr,
+    google_dataplex_asset.gcp_primary_ga4_obfuscated_sample_ecommerce,
+    google_dataplex_asset.gcp_primary_tables
+  ]
+}
+
+# # wait for completion
+resource "time_sleep" "wait_after_all_workflows" {
```
Comment (davenportjw) [inline workflows.tf:140]: do we need this resource if we are waiting on the other waiter for completion? want to make sure we dont doublecount

Comment (bradmiro) [reply workflows.tf]: Removed.

**Review by davenportjw — APPROVED** [review]

**Pull request merged**